code/src/pages/MovieDetails.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useEffect} from 'react'
+import { useParams, Link } from 'react-router-dom'
+
+import BackArrow from '../components/icons8-back-arrow-64.png';
```

Comment:
Uppercase naming is reserved for components only, not images. it should rather be :

```
import backArrow from '../components/icons8-back-arrow-64.png';
```